packages/webchannel-wrapper/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # @firebase/webchannel-wrapper
 
+## 0.10.7
```

Comment:
Don't manually update this file, it will be automatically updated with the release using the changesets tool.